Cargo.toml

```diff
@@ -1,11 +1,12 @@
 [package]
 name = "polars-tdigest"
-version = "0.1.8"
+version = "0.1.9"
 edition = "2021"
 
 [lib]
 name = "polars_tdigest"
-crate-type= ["cdylib"]
+crate-type= ["rlib"]
```

Comment:
had to change this in order to be able to run benchmarks

Reply:
I do not think, that it will work by building a plugin:

```
--crate-type=cdylib, #![crate_type = "cdylib"] - A dynamic system library will be produced. This is used when compiling a dynamic library to be loaded from another language. This output type will create *.so files on Linux, *.dylib files on macOS, and *.dll files on Windows.
--crate-type=rlib, #![crate_type = "rlib"] - A “Rust library” file will be produced. This is used as an intermediate artifact and can be thought of as a “static Rust library”. These rlib files, unlike staticlib files, are interpreted by the compiler in future linkage. This essentially means that rustc will look for metadata in rlib files like it looks for metadata in dynamic libraries. This form of output is used to produce statically linked executables as well as staticlib outputs.
```

As we nowhere build again and load and register plugin dynamically from Python interpreter. But we decided to check this practically.